.circleci/config.yml

```diff
@@ -32,3 +32,14 @@ jobs:
       - run:
           name: Launch build with bazel
           command: echo "bazel build //upload --verbose_failures" | HOMEWORLD_CHROOT="$HOME/autobuild-chroot" USER="circleci" ./build-chroot/enter-ci.sh
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build:
+          filters:
+            branches:
+              only:
+                - staging
+                - trying
+                - master
```

Comment:
Do we want to apply this same configuration to Jenkins?

Reply:
I'm not sure how to do that - I think it requires configuring jenkins through the web interface?

Reply:
This is now working; it's configured in jenkins itself.